packages/frontend/src/commons/index.ts

```diff
@@ -90,4 +91,9 @@ export const AMM_SUPPORTS_RANGE_INCENTIVES: Record<SupportedAmm, boolean> = {
     [SupportedAmm.StableSwap]: false,
 };
 
+// taken from https://github.com/wevm/wagmi/blob/80326815bea2f175623157f57465f9dfae1f4c5c/packages/connectors/src/safe.ts#L45
+export const SAFE_CONNECTOR_ID = "safe";
+
 export const metromApiClient = METROM_API_CLIENT[ENVIRONMENT];
+
+export const safeSdk = new SafeAppsSdk();
```

Comment:
Can we call this `SAFE_APP_SDK`? At this point also `METROM_API_CLIENT`

Reply:
Addressed in fac464cc809048acf847346fb83510fe75679d46.